contracts/LifCrowdsale.sol

```diff
@@ -22,7 +23,21 @@ contract LifCrowdsale is Ownable, Pausable {
 
   // address where funds are collected
   address public foundationWallet;
-  address public marketMaker;
+
+  // minimun amount of wei to be raised in order to succed, it starts in USD
+  uint256 public maxCapPresaleUSD = 1000000;
+
+  // minimun amount of wei to be raised in order to succed, it starts in USD
+  uint256 public minCapUSD = 5000000;
+
+  // maximun balance that the foundation can have, it starts in USD
+  uint256 public foundationBalanceCapUSD = 10000000;
+
+  // maximun balance that the 24 month market maker can have
+  uint256 public marketMaker24PeriodsCapUSD = 40000000;
+
+  // how much a USD worth in wei
+  uint256 public weiPerUSD = 0;
```

Comment:
We need two of these variables: one for the public presale and one for the TGE

Reply:
We really need two variables? I think one is enough, it can be changed X amount of blocks before publicPresale and ICO, on the TGE it will use the last value passed before the ICO.

Reply:
I think it's better. It's definitely better to understand if they have clear names whether they refer to presale or TGE. Also imagine if the TGE starts right after the presale, then by updating the value for the TGE we are changing the value for the presale while it is running. I think the safest is to keep them separate